### DISCUSSION TITLE
Eror knitting to HTML

### DISCUSSION THREAD

**Discussion by rachmat-kimung** [teamalgoritma/community#358] [Q&A] [answered]
mengalami eror pada saat knitting to HTML "Eror while opening file" --> The system cannot find the file specified
dibagian render tertulis error in py_call_impl(callable, dot$args, dots$keywords) : ImportError: could not import PIL.IMAGE. The use of load_img .....
![image](https://user-images.githubusercontent.com/79957356/192676572-d19040b6-6578-439b-8fed-f482d38e4dfc.png)

**Comment by fafilia** [comment, marked as answer]
Dear @rachmat-kimung ,
Terkait kendala yang Anda hadapi, silahkan cek pada pertanyaan berikut:  https://github.com/teamalgoritma/community/discussions/321 karena sepertinya errornya mirip. Mungkin boleh dilampirkan juga line yang membuat proses knitting error(di screenshot ada keterangan baris ke 321).

**Reply by rachmat-kimung** [reply]
untuk refrence diatas sudah saya lakukan dan tetap eror, line eror terlampir

![image](https://user-images.githubusercontent.com/79957356/192751753-99296c51-ac72-49b7-b8ca-f881b6d52641.png)

**Reply by rachmat-kimung** [reply]
sudah solve, yg saya lakukan pip uninstall pillow di prompt anaconda kemudian instal lagi pip install pillow di prompt anaconda juga
